package.json

```diff
@@ -132,6 +132,9 @@
       "optional": true
     }
   },
+  "resolutions": {
+    "ember-get-config": "0.5.0"
+  },
```

Comment:
Please remove this resolution after `ember-cli-addon-docs` makes a new release, in which `ember-get-config` is no longer a dependency.